lib/rules/selector-pseudo-class-case/__tests__/index.js

```diff
@@ -114,120 +115,161 @@ testRule(rule, {
   reject: [
     {
       code: "a:Hover { color: pink; }",
+      fixed: "a:hover { color: pink; }",
       message: messages.expected(":Hover", ":hover"),
       line: 1,
       column: 2
     },
+    {
+      code: "/* comment */ a:Hover { color: pink; }",
+      fixed: "/* comment */ a:hover { color: pink; }",
+      message: messages.expected(":Hover", ":hover"),
+      line: 1,
+      column: 16
+    },
+    {
+      code: "a:Hover /* comment */ { color: pink; }",
+      fixed: "a:hover /* comment */ { color: pink; }",
+      message: messages.expected(":Hover", ":hover"),
+      line: 1,
+      column: 2
+    },
```

Comment:
Could you add the following test cases?

`a:Hover,/*comment*/ .b {color: pink;}`

Reply:
Hi, thank you very much for the example! I have updated the PR